packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/modeledDataStreamQueryEditor/modeledDataStreamExplorer/modeledDataStreamTable/modeledDataStreamTable.tsx

```diff
@@ -153,13 +160,19 @@ export function ModeledDataStreamTable({
       }
       preferences={<ModeledDataStreamTablePreferences preferences={preferences} updatePreferences={setPreferences} />}
       ariaLabels={{
-        itemSelectionLabel: (isNotSelected, modeledDataStream) =>
-          isNotSelected
-            ? `Select modeled data stream ${modeledDataStream.name}`
-            : `Deselect modeled data stream ${modeledDataStream.name}`,
+        itemSelectionLabel: ({ selectedItems }, modeledDataStream) => {
```

Comment:
probably worth moving to a utility function given this will grow in logic when we add validation for all widget types

Reply:
changes updated